scripts/run-cloud-server.py

```diff
@@ -0,0 +1,90 @@
+import boto3
+import os
+import argparse
+
+sts = boto3.client('sts')
+
+project_name = os.environ.get('TF_VAR_project_name', 'Support Sphere').lower().strip().replace(" ", "")
+neighborhood = os.environ.get('TF_VAR_neighborhood', 'Laurelhurst').lower().strip().replace(" ", "")
```

Comment:
nit: maybe have a `sanitize` function that does the lower, strip, and replace.

Reply:
okay that's a much better way to do this haha, can do!